src/app/pages/PerpetualPage/contexts/GraphQLProvider.tsx

```diff
@@ -128,13 +123,14 @@ const config: GraphQLEndpoint[] = [
   },
 ];
 
-const RETRY_DELAY = 2 * 60 * 1000;
+const RETRY_DELAY = 2 * 60 * 1000; // 2 seconds
```

Comment:
This should be 2 minutes
```suggestion
const RETRY_DELAY = 2 * 60 * 1000; // 2 minutes
```